src/cache.js

```diff
@@ -0,0 +1,124 @@
+import logger from './logger';
+
+/**
+ * @description A module to handle caching data from webex sdk.
+ *
+ * @module CacheMeOutside No-pun intented
+ * @returns {object} API methods
+ */
+export function CacheMeOutside() {
```

Comment:
Is there a reason to not use the `class` format like the rest of the modules?

Reply:
Could do that